CMakeLists.txt

```diff
@@ -158,12 +164,76 @@ if(OCOS_ENABLE_SELECTED_OPLIST)
   include(_selectedoplist)
 endif()
 
+set(_OCOS_EXCEPTIONS_REQUIRED OFF)
+if (OCOS_ENABLE_GPT2_TOKENIZER OR
+    OCOS_ENABLE_WORDPIECE_TOKENIZER OR
+    OCOS_ENABLE_BLINGFIRE OR
+    OCOS_ENABLE_SPM_TOKENIZER OR
+    (OCOS_ENABLE_CV2 OR OCOS_ENABLE_OPENCV_CODECS OR OCOS_ENABLE_VISION))
+    set(_OCOS_EXCEPTIONS_REQUIRED ON)
+endif()
+
+# Special case an embedded build with ORT exceptions disabled but custom ops that require exceptions.
+# Allow using an override so we can do a direct build of ort-ext in a CI without having to embed it in an ORT build.
+set(_OCOS_PREVENT_EXCEPTION_PROPAGATION OFF)
+if (_OCOS_PREVENT_EXCEPTION_PROPAGATION_OVERRIDE)
```

Comment:
This allows building with _OCOS_PREVENT_EXCEPTION_PROPAGATION_OVERRIDE=ON to simulate an embedded build with exceptions disabled so that the test_exceptions.cc tests can be run. Will look at setting up a CI with this tomorrow unless there's a reason to prefer a full embedded build in the CI.